bindings/python/multibody/joint/joints-datas.hpp

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright (c) 2019 CNRS
+//
+
+#ifndef __pinocchio_python_joints_datas_hpp__
+#define __pinocchio_python_joints_datas_hpp__
+
+#include <eigenpy/exception.hpp>
```

Comment:
They are not needed in fact. Only boos/python.hpp is requested here

Reply:
Done.